chain/chain/Cargo.toml

```diff
@@ -36,8 +37,8 @@ tracing.workspace = true
 near-async.workspace = true
 near-cache.workspace = true
 near-chain-configs = { workspace = true, features = [
-  "test_genesis",
-  "test_utils",
+    "test_genesis",
+    "test_utils",
```

Comment:
THB I'm not a fan of whitespace changes. Can you update your config to remove those or are they inconsistent?

Reply:
I've configured my IDE to use 4-space indentation for `.toml` files, so it automatically re-formats every file I edit. I believe it's a step in the right direction (to have consistent 4-space indentation everywhere).